idd/Energy+.idd.in

```diff
@@ -105255,6 +105255,22 @@ OutputControl:Files,
        \key No
        \default Yes
 
+OutputControl:Timestamp,
+       \memo Control timestamp format, currently applies only to JSON and native CSV (not CSV via ReadVars)
```

Comment:
I think this is clear enough.  And I'm glad this object defaults to the current state.  Hopefully this soft-opening gives a few users a chance to not only test out the implementation, but also move away from the ReadVars based output workflow.